docs/monthly-meeting/2023-05.md

```diff
@@ -0,0 +1,121 @@
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+
+## Roll call
+
+(Name / `@GitHubUsername`)
+- Mariatta `@Mariatta`
+- Petr Viktorin `@encukou`
+- Cristián Maureira-Fredes `@cmaureir`
+- Hugo van Kemenade`@hugovk`
+- Jim DeLaHunt `@JDLH`
+- Ryan Duve `@ryan-duve`
+- Joanna Jablonski `@jablonskidev`
+- Ezio Melotti `@ezio-melotti`
+- Keto Zhang `@ketozhang`
+- Manuel Kaufmann `@humitos`
+- Ned Batchelder `@nedbat`
+- C.A.M. Gerlach `@CAM-Gerlach`
+- Marcus Sherman `@betteridiot`
+- Yaa Nuamah Kusi-Fordjour `@`
+
+
+## Introductions
+
+> If there are any new people, we should do a round of introductions.
+
+- Petr, Python core dev, runs these meetings
+- Mariatta, one of the people who started the Docs group. Just chaired Pycon US
+- Hugo, from Helsinki Python core dev
+- Ryan Duve, data scientist, Python user interested helping with community docs stuff
+- Joanna, Python education
+- Jim, from Vancouver, Canada; reads docs & fixes things he notices
+- Manuel Kaufmann from Argentina now in Spain, translations, Read the Docs, CPython translation in Spanish, PSF Fellow
+- Marcus Sherman, from Portland, Maine @ Northeastern University, `@betteridiot`, biologist by training
+- Ned, from Boston, made the docs official, eager to see this group grow into a large & welcoming community
+- Cristián Maureira-Fredes, CPython Spanish translation
+- Keto Zhang, software engineer in astronomy, went to Cam's talk at PyCon
+- CAM, gave the talk at PyCon, new Python core dev, from Huntsville, Alabama, Spyder maintainer, looking to be a full-time open-source dev & documentarian
+- Ezio, core Dev, working on Docs and infrastucture
+
+
+## Reports and celebrations
+
+> 60 second updates on things you have been up to, questions you have, or developments you think people should know about. Please add yourself, and if you do not have an update to share, you can pass.
+
+> This is a place to make announcements (without a need for discussion). This is also a great place to give shout-outs to contributors! We'll read through these at the beginning of the meeting.
+
+Congrats to CAM!
+
+Congrats Ned for the [keynote at PyCon US 2023](https://us.pycon.org/2023/about/keynote-speakers/#ned-batchelder)
+
+
+* [PyCon](https://us.pycon.org/2023/), April 19-27 2023:
+  * CAM & Erlend's talk, "[Iteration toward Transformation of the Python Documentation](https://us.pycon.org/2023/schedule/presentation/88/)", went well, good discussions afterwards and many joined the Discord wanting to contribute
+  * Sprints: some docs contributors, including people making their very first CPython contribution
+  * Sprints: [Hugo] switched CPython docs previews from Netlify to RtD.
+    * Main benefit: multiple admins instead of single-person bottleneck with Netlify.
+    * https://github.com/python/cpython/pull/103843
```

Comment:
This and some below need manually linkifying:

```suggestion
    * [https://github.com/python/cpython/pull/103843](https://github.com/python/cpython/pull/103843)
```

Reply:
Oops, seems like I missed that one. Using shorter and more readable link text:

```suggestion
    * [python/cpython#103843](https://github.com/python/cpython/pull/103843)
```